nx.json

```diff
@@ -74,9 +83,9 @@
     }
   },
   "plugins": [
-    "./examples/e2e/cli-e2e-original/tooling/original.plugin.ts",
     {
-      "plugin": "./dist/projects/nx-verdaccio/src/index.js",
+      "plugin": "./projects/nx-verdaccio/src/plugin/nx-verdaccio.plugin.ts",
+      "include": ["examples/projects/**", "examples/e2e/**"],
```

Comment:
Is this necessary? I guess we don't need to include or exclude and folder or project from the plugin in this repository.

Reply:
If we keep the original plugin, it would make sense to exclude `examples/e2e/cli-e2e-original/` at least

Reply:
The intent was mainly to demonstrate a best practice to only run the plugin where needed. This `nx-verdaccio` plugin is catching all Nx projects (at least all those having a project.json file) and it comes at a cost.

Reply:
Also thought of an early exit, but it would still rely on reading the project configuration right?
We should probably do both anyway.